humility-core/src/reflect.rs

```diff
@@ -564,6 +588,15 @@ impl Format for Tuple {
     ) -> Result<()> {
         fmt.indent += 4;
 
+        // Is this a bitflags-generated type? If so, just format it as a binary
+        // value.
+        if self.name().contains("InternalBitFlags") {
+            if let Some(flags) = self.1.get(0).and_then(|v| v.as_base().ok()) {
+                write!(out, "{flags:#b}")?;
```

Comment:
perhaps we should always format the type with every zero bit included? I could be convinced either way on that.

Reply:
I could go either way too -- but I think it looks as is though!